README.md

```diff
@@ -557,6 +558,66 @@ For detailed documentation on the InstructLab LLMs and their functions, see the
    ilab model chat --model ~/.cache/instructlab/models/mixtral-8x7b-instruct-v0.1.Q4_K_M.gguf --temperature 0.5
    ```
 
+### 📇 Configure retrieval-augmented generation (developer preview)
+
+If you want to use the developer preview of retrieval-augmented generation (RAG), you need to enable developer preview features by setting `ILAB_FEATURE_SCOPE` to `DevPreviewNoUpgrade`. This tells InstructLab to enable developer preview features that might not be upgradable.  For many popular shells, this is done via the following:
+
+   ```shell
+   export ILAB_FEATURE_SCOPE=DevPreviewNoUpgrade
+   ```
+
```

Comment:
seems it would better to mention to download the embedding model..

Reply:
Yes, this is very important.  I will do that.  Thanks!